http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/WsmDao.scala

```diff
@@ -379,16 +406,47 @@ object WsmDecoders {
     )
   }
 
+  implicit val landingZoneDecoder: Decoder[LandingZone] =
+    Decoder.forProduct5("landingZoneId", "billingProfileId", "definition", "version", "createdDate")(LandingZone.apply)
+  implicit val listLandingZonesResultDecoder: Decoder[ListLandingZonesResult] =
+    Decoder.forProduct1("landingZones")(ListLandingZonesResult.apply)
+
+  implicit val landingZoneEncoder: Encoder[LandingZone] =
```

Comment:
I think all these are in the `WsmDecoders` object, so probably shouldn't have encoders.

Do we even need encoders for these objects, since I think `HttpWsmDAO` is only decoding them from GET requests?

Reply:
Oh! I see. I've moved them down to the WsmEncoders section in the upcoming revision. They're used for unit testing the HttpWsmDao implementations-